vendor/gopkg.in/raintank/schema.v1/metric.go

```diff
@@ -113,7 +112,7 @@ type MetricDataArray []*MetricData
 
 // for ES
 type MetricDefinition struct {
-	Id       string `json:"id"`
+	Id       MKey   `json:"id"`
 	OrgId    int    `json:"org_id"`
 	Name     string `json:"name" elastic:"type:string,index:not_analyzed"` // graphite format
 	Metric   string `json:"metric"`                                        // kairosdb format (like graphite, but not including some tags)
```

Comment:
i would really like to get rid of "Metric" it is currently always set to the same value as "Name"

Reply:
We should also update SetId() to use m.Name instead of m.Metric. and remove the Validate() rule requiring it.
We can also safely drop the field from the metricData payload.  If clients continue to send it, it will just be ignored when marshaling the data.

Reply:
@woodsaj and same for the index right? I'll update the code to remove the property from the structures and basically ignore that field in cassandra.